client/src/test/java/io/pravega/client/batch/impl/BatchClientImplTest.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Revert format changes to license header.

Reply:
Done